server/src/recipe/recipe.controller.js

```diff
@@ -34,4 +34,15 @@ const listUserRecipes = async (req, res, next) => {
   }
 };
 
-export { listPublishedRecipes, listUserRecipes };
+const newRecipe = async (req, res, next) => {
+  try {
+    const recipe = await Recipe.create(req.body);
+
+    res.json({ recipeId: recipe.id });
+  } catch (err) {
+    // TODO: maybe catch more specific error
+    next(new Error());
+  }
+};
```

Comment:
not to sound harsh but this function doesn't really bring much to the table here as it is. I think it's better to take some time and do things properly with their implementation when you need to then to do it ahead of time and than backtrack and change things and rediscover what you've done. For now it can stay but in the future 99% chance this function is going to change because of validation, user login checking etc while this had no real value for now